src/providers/sources/remotestream.ts

```diff
@@ -2,7 +2,7 @@ import { flags } from '@/entrypoint/utils/targets';
 import { makeSourcerer } from '@/providers/base';
 import { NotFoundError } from '@/utils/errors';
 
-const remotestreamBase = `https://fsa.remotestre.am`;
+const remotestreamBase = atob('aHR0cHM6Ly9mc2IuOG1ldDNkdGpmcmNxY2hjb25xcGtsd3hzeGIyb2N1bWMuc3RyZWFt');
```

Comment:
Just run it, why you keep it like this?

```suggestion
const remotestreamBase = 'https://fsb.8met3dtjfrcqchconqpklwxsxb2ocumc.stream';
```

Reply:
nuh uh